<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomMask</class>
 <widget class="QDialog" name="RandomMask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Randomly Data Selection</string>
  </property>
  <widget class="QGroupBox" name="groupBox_randMask_inputs">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>220</width>
     <height>220</height>
    </rect>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBox_randMask">
        <item>
         <property name="text">
          <string>2D Image</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3D Image</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4D Image</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_randMask_percentage">
          <property name="text">
           <string>Percentage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_randMask_percentage"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_randMask_imgUpl">
        <property name="text">
         <string>Open Image Data</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_randMask_run">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_randMaks_ProcImg">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>420</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_randMask_origImg">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>540</y>
     <width>390</width>
     <height>390</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>850</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_randMask_fileName">
      <property name="text">
       <string>File Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_randMask_fileName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_randMask_fileNameSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="horizontalSlider_radnMaskProcImg">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>550</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_randMaskOrigImg">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>950</y>
     <width>370</width>
     <height>17</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
